src/browser/appwindow.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
+var _ = require('underscore-plus');
```

Comment:
Please make sure there are no unnecessary lines movement like this in the PR. Thanks